### PR TITLE
MAG-5 Create new theme;

### PR DESCRIPTION
**Pull request by YevheniiH**
Create new theme

### CODE REVIEW

File: app/design/frontend/Jungle-luma/jungle_theme/composer.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "AFL-3.0"
+    ],
+    "config": {
+        "sort-packages": true
+    },
+    "version": "100.4.5",
+    "require": {
+        "php": "~7.4.0||~8.1.0",
+        "magento/framework": "103.0.*",
+        "magento/theme-frontend-blank": "100.4.*"
```
Comment (AndriiBatoh) [inline app/design/frontend/Jungle-luma/jungle_theme/composer.json]: Your theme is inherited from luma, why did you add dependency to the blank, change it

File: app/design/frontend/Jungle-luma/jungle_theme/etc/view.xml [hunk 2]
```diff
@@ -0,0 +1,314 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
```
Comment (AndriiBatoh) [inline app/design/frontend/Jungle-luma/jungle_theme/etc/view.xml]: When you copy default magento files, coupyright comment block should be removed, remove it please

File: app/design/frontend/Jungle-luma/jungle_theme/registration.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
```
Comment (AndriiBatoh) [inline app/design/frontend/Jungle-luma/jungle_theme/registration.php:2]: Remove copyright block

File: app/design/frontend/Jungle-luma/jungle_theme/theme.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
```
Comment (AndriiBatoh) [inline app/design/frontend/Jungle-luma/jungle_theme/theme.xml]: Remove copyright block add theme preview image

File: dev/tools/grunt/configs/local-themes.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
+/**
```
Comment (AndriiBatoh) [inline dev/tools/grunt/configs/local-themes.js]: Config files related to NPM, GRUNT, COMPOSER, and others should not be pushed in the commit remove this file from here

**Review by AndriiBatoh — CHANGES_REQUESTED** [review]

File: app/design/frontend/Jungle-luma/jungle_theme/composer.json (hunk 1 above)
Comment (YevheniiH) [reply app/design/frontend/Jungle-luma/jungle_theme/composer.json]: 17. made changes to "magento/theme-frontend-luma".

File: app/design/frontend/Jungle-luma/jungle_theme/etc/view.xml (hunk 2 above)
Comment (YevheniiH) [reply app/design/frontend/Jungle-luma/jungle_theme/etc/view.xml]: Removed the comment block.

File: app/design/frontend/Jungle-luma/jungle_theme/registration.php (hunk 3 above)
Comment (YevheniiH) [reply app/design/frontend/Jungle-luma/jungle_theme/registration.php:2]: Removed the comment block.

File: app/design/frontend/Jungle-luma/jungle_theme/theme.xml (hunk 4 above)
Comment (YevheniiH) [reply app/design/frontend/Jungle-luma/jungle_theme/theme.xml]: Removed the comment block.
